server.js

```diff
@@ -28,10 +28,5 @@ app.use(express.static(path.join(__dirname, BUILDDIR)));
 var server = app.listen(PORT, () => {
     console.log(`\nTemp server started at http://localhost:${PORT}!`);
     //console.log('press ctrl+c to stop ');
-
-    var opn = require('open');
```

Comment:
FYI: I will probably add another PR to bring back autoopen, this time as script run in parallel to `hot` script 👍

Reply:
Resolved with commit 8fbe48e

Reply:
Should be addressed in commit 2771508